tests/functional/pages/surveys/census/communal/establishment-type.page.js

```diff
@@ -33,6 +33,11 @@ class EstablishmentTypePage extends MultipleChoiceWithOtherPage {
     return this
   }
 
+  setEstablishmentTypeAnswerOtherOtherText(value) {
```

Comment:
But if the text isn't Other it would read something like `setEstablishmentTypeAnswerHotelText`?